spec/unit/puppet/provider/chocolateysource/windows_spec.rb

```diff
@@ -444,6 +489,26 @@
       resource.flush
     end
 
+    it "should set bypass_proxy when present" do
```

Comment:
What's the difference between this test and the one starting at line 596?

Reply:
Functionally none, I don't think, this is an artifact of copying the tests over.